src/graphql/types.ts

```diff
@@ -30,7 +30,6 @@ export type Colony = {
   founder?: Maybe<User>;
   isNativeTokenExternal: Scalars['Boolean'];
   nativeTokenAddress: Scalars['String'];
-  programs: Array<Program>;
```

Comment:
Please don't touch this file manually. This is generated automatically.

So please restore this to it's original version, then run `npm run graphql:typegen` and commit the resulting file